### ISSUE TITLE
New Feature Suggestions for FoE-Extender

### ISSUE THREAD

**Issue by Zundarian** [cosmos0nash/FoE-Extender#5]
Hello,

I've been using the FoE-Extender and I have some suggestions for new features that I believe would greatly enhance the user experience and functionality of the tool. Here they are:

1. **Automatic Collection from Legendary Buildings, Town Hall, and Other Selected Buildings:** I would love to see a feature where the tool can automatically collect from Legendary Buildings, the Town Hall, and other buildings of the user's choice. This would save a lot of time and make the game more efficient.

2. **Automatic Collection of Chests in the Castle System:** Another feature that would be very useful is the automatic collection of chests in the castle system. This would also save a lot of time.

3. **Automatic Investment of Forge Points:** Lastly, I think it would be great if there was a feature that automatically invests all available Forge Points into research when activated. This would ensure that no Forge Points go to waste.

I believe these features would greatly improve the FOE-Extender and make it even more useful for players. I hope you will consider implementing them.

Thank you for your time and consideration.

**Comment by cosmos0nash** [comment]
> ciao cosmos sono 3 mesi che rinnovo volevo chiederti se potevi aggiungere una cosa che ti permette di mettere gli assedi non da ban tipo di aggiungere la velocita di assedio tipo come le battaglie da 0 a 1000 millisecondi ti è possibile farlo che cosi ti permette di esser invisibile

Of course I will add it as soon as possible

**Comment by madmaxdx** [comment]
grazie grandissimo

**Comment by madmaxdx** [comment]
I saw that you put the siege speed I would like to understand that speed what siege changes you because if I click on place siege army starts very fast

**Comment by cosmos0nash** [comment]
> I saw that you put the siege speed I would like to understand that speed what siege changes you because if I click on place siege army starts very fast

I don’t understand why there is a delay when the button is pressed, you can wait yourself before pressing the button. a delay is made to automatically deploy an army during an attack or when the "auto re-siege" option is enabled

**Comment by madmaxdx** [comment]
ok tanks

**Comment by madmaxdx** [comment]
hello but an autopilot for the gvg?

**Comment by madmaxdx** [comment]
?????

**Comment by madmaxdx** [comment]
fast defenses will you add them to counter bots against and put defenses?

**Comment by JayseeXD** [comment]
Is there any plan for sniping?

**Comment by RiseMe-origami** [comment]
Automatic GB sniping in neighbourhood, friends and guild would be game changer

**Comment by JayseeXD** [comment]
Yeah it's just math, unfortunately author said this is not gonna be a thing via e-mail sorry :(

**Comment by RiseMe-origami** [comment]
Maybe automatic reoccuring quest completing (the one where you pay gold and supplies)

**Comment by tigertim08** [comment]
Would be great if it would be possible in GB to set the number of attacks based on the attrition. For example
Up to 24%, 5 fights 
Up to 48%, 10 fights etc.

**Comment by Snow1234Snow** [comment]
Would it be possible to interrupt the BG bot after it has lost a higher age unit and heal it with diamonds before it loses the higher age units for real?

**Comment by FoEKingBro** [comment]
Can you add a feature where the Terminals (AF and OF) can be collected and automatically selects the crew, heals the ships and sends them out. Also the Space Age sites (for special goods), 2 options where it collects via Negs or Attacks for each site. Also after collection, restarts the cycle again on that site. Doing this would help save so much time and alot of forgetfulness

**Comment by RiseMe-origami** [comment]
Some automation for QI would be a gamechanger (not only conpleting nodes but also building the city)

**Comment by paperboy11** [comment]
city building on QI or auto donate  would be sick. maybe auto fighting too. really think if we could get rivals and  those quests auto it would be sick
